### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we performed :

- Build interactive console apps.
- Run a program using the command line.

### CODE REVIEW

**Review by DammyShittu — COMMENTED** [review]
Hi @AnicetFantomas,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

#### You have submitted a Pull Request with failing linters. Kindly fix them

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    me = nil
+    print "\nID of person: "
+    person_id = gets.chomp
+    @people.each do |x|
+      me = x if x.id == person_id.to_i
+    end
+    me.rentals.each do |x|
+      puts x.date
+    end
+    puts
```
Comment (codepantha) [inline app.rb]: - [ ] Alongside the rental date, please also show the rental book name and author 🙏

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    @my_books.each_with_index do |x, index|
+      puts "#{index}) Title: \"#{x.title}\", Author: #{x.author} "
+    end
+  end
+
+  def list_people
+    @people.each_with_index do |x, index|
+      puts "#{index}) [#{x.class.name}] Name: #{x.name}, ID: #{x.id}, Age: #{x.age}"
+    end
+  end
```
Comment (codepantha) [inline app.rb:58]: - [ ] The person name does not display correctly as shown in the console image below 👇 Please fix it
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUESTED CHANGES ♻️ 

## Hi @AnicetFantomas 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights 🎯 
✔️ Descriptive PR
✔️ Professional README
✔️ Linter checks

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      action_list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rental
+    else
```
Comment (BenMukebo) [inline app.rb:42]: ### 👏🏻 Great job Everything works fine 👍🏻 💯 Congrat 🍾 

great

**Review by BenMukebo — APPROVED** [review]
### Hi @ , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**